src/api/app/components/notification_avatars_component.rb

```diff
@@ -20,6 +20,8 @@ def avatar_objects
                           [User.find(@notification.event_payload['user_id'])]
                         when 'Decision'
                           [User.find(@notification.event_payload['moderator_id'])]
+                        when 'WorkflowRun'
+                          [Token.find(@notification.event_payload['token_id'])&.executor]
```

Comment:
```suggestion
                          [Token.find(@notification.event_payload['token_id'])&.executor].compact
```
In order to remove the nil that will appear if there is no executor or no token